R/tuneParams.R

```diff
@@ -104,7 +104,7 @@ tuneParams = function(learner, task, resampling, measures, par.set, control, sho
   opt.path = makeOptPathDFFromMeasures(par.set, measures, include.extra = (control$tune.threshold))
   if (show.info) {
     messagef("[Tune] Started tuning learner %s for parameter set:", learner$id)
-    messagef(printToChar(par.set))
+    message(printToChar(par.set))
```

Comment:
Please add a comment here, otherwise people might think of changing it back.

Otherwise this looks good. If this problem occours mutiple times we can also fix this in BBmisc upstream